#57: Fixed release build

tomuben opened the pull request:
Added needed actions to release IAM role.
fixes #57

tomuben committed:
#57: Fixed release build

tomuben committed:
replace s3 actions

tkilias commented on exasol_script_languages_developer_sandbox/templates/release_code_build.jinja.yaml:
```diff
@@ -68,6 +68,8 @@ Resources:
                 Effect: Allow
               - Action:
                 - s3:PutObject
+                - s3:GetObject
```
I guess this for the rename, maybe add this as comment

tomuben replied:
I kind of inverted the documentation: I added all necessary actions to the `AwsAccess` class. I think that's better and more clear.

tomuben replied:
With  that, one can do a full text search for "s3:GetObject" in the project and will find the API.

tomuben committed:
1. documented required actions in AwsAccess API
2. removed unnecessary actions from release_code_build.jinja.yaml
3. added needed action
